src/lightning_app/cli/lightning_cli.py

```diff
@@ -206,15 +290,63 @@ def stop():
     pass
 
 
-@_main.group(hidden=True)
+@_main.group(name="delete")
 def delete():
-    """Delete an application."""
+    """Delete Lightning AI BYOC managed resources."""
     pass
 
 
-@_main.group(name="list", hidden=True)
+@delete.command("cluster")
+@click.argument("cluster", type=str)
+@click.option(
+    "--force",
+    "force",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="""Delete a BYOC cluster from Lightning AI. This does NOT delete any resources created by the cluster,
+            it just removes the entry from Lightning AI.
+
+            WARNING: You should NOT use this under normal circumstances.""",
+)
+@click.option(
+    "--wait",
+    "wait",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="Enabling this flag makes the CLI wait until the cluster is deleted.",
+)
+def delete_cluster(cluster: str, force: bool = False, wait: bool = False):
+    """Delete a Lightning AI BYOC compute cluster and all associated cloud provider resources.
+
+    Deleting a run also deletes all Runs and Experiments that were started on the cluster.
```

Comment:
This is out-dated right ? Would Experiments and Runs be available on the cluster?

Reply:
it is - I've fixed the doc string.